contracts/TimedAssetLockContract/Deployment.py

```diff
@@ -0,0 +1,56 @@
+from algosdk.future import transaction
+from algosdk import account, mnemonic, logic
+
+import AkitaInuASAUtils as AIASAU

```

Comment:
Maybe we could have

from AkitaInuASAUtils import *

And just remove the AIASAU from the calls